.github/workflows/go-ci.yaml

```
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.18.2-buster]
+        go-version: [1.23.9-bookworm]
```

nit: I'm not sure why do we specify patch version nor debian version here:
```suggestion
        go-version: ["1.23"]
```